test/dotnet-trx-merge.IntegrationTests/Fixtures/MergeWithMyFiles/_fv-az635-124_2023-10-04_11_32_15.trx

```diff
@@ -0,0 +1,1408 @@
+    <UnitTestResult executionId="51efa09d-1d66-46df-a12e-3bfe77c22610" testId="164a2c5b-e354-ac1a-d0ca-0f22934e0f13" testName="SavedCardNoUserDetail" computerName="fv-az635-124" duration="00:00:00.0012669" startTime="2023-10-04T11:32:16.3593600+00:00" endTime="2023-10-04T11:32:16.3619630+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="51efa09d-1d66-46df-a12e-3bfe77c22610" />
+    <UnitTestResult executionId="3dcbf370-32ca-4e9d-9adb-6347ff0b2711" testId="a70a19ba-1b6e-12f7-fa00-2c945790e66d" testName="ShouldReturnStatusUnknownAndAMessageWhenProviderResponseIsNull" computerName="fv-az635-124" duration="00:00:00.0019561" startTime="2023-10-04T11:32:16.5673890+00:00" endTime="2023-10-04T11:32:16.5700450+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="3dcbf370-32ca-4e9d-9adb-6347ff0b2711" />
+    <UnitTestResult executionId="8600498c-e1f2-4379-826f-73588677418a" testId="7edc5100-eca1-470d-d2ed-037ada91af38" testName="ShouldReturnStatusUnknownAndAMessageWhenProviderReturnsErrors" computerName="fv-az635-124" duration="00:00:00.0018083" startTime="2023-10-04T11:32:16.5743860+00:00" endTime="2023-10-04T11:32:16.5769130+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="8600498c-e1f2-4379-826f-73588677418a" />
+    <UnitTestResult executionId="c8c636c5-8d1a-40d1-9b20-65d968042613" testId="7f021ba4-ce47-c759-0e9a-e85913d720db" testName="ShouldReturnCorrectTransactionStatus (BLOCKED,CompleteFailed,Optimis.SmartPayBureau.Client.error[])" computerName="fv-az635-124" duration="00:00:00.0000078" startTime="2023-10-04T11:32:16.6593360+00:00" endTime="2023-10-04T11:32:16.6600670+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="c8c636c5-8d1a-40d1-9b20-65d968042613" />
+    <UnitTestResult executionId="868217d3-4a8f-4036-9886-314c10b3f4b9" testId="bd0f3e87-2d49-7223-2c00-e0924f9c83f5" testName="SubsequentRecurringCardNoUserDetail" computerName="fv-az635-124" duration="00:00:00.0006997" startTime="2023-10-04T11:32:16.3624740+00:00" endTime="2023-10-04T11:32:16.4115700+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="868217d3-4a8f-4036-9886-314c10b3f4b9" />
```

Comment:
if you search for this test result in the file: bd0f3e87-2d49-7223-2c00-e0924f9c83f5 you'll find that it is duplicated quite 8 times